vrc/loaders/cparser.cc

```diff
@@ -134,6 +196,67 @@ enum CXChildVisitResult visit_function_body(CXCursor c, CXCursor parent, Visitor
     return result;
 }
 
+void retrieve_annotations(CXCursor c, VisitorState *state, size_t node_i)
+{
+    functor_visit(c, [state, node_i](CXCursor c, CXCursor parent) {
+        if (c.kind == CXCursor_AnnotateAttr) {
```

Comment:
Why not reuse `add_label`?

Reply:
Because it attributes the annotations to `state->current_function`.